scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -438,6 +438,7 @@ class Router(formatOps: FormatOps) {
             // The block will take care of indenting by 2.
             Seq(Split(Space, 0))
           case _ =>
+            val rhsIsComment = isSingleLineComment(right)
             Seq(
               Split(
```

Comment:
What happens if we `ignoreIf = rhsIsComment` on this split? I'm a bit uncomfortable with having three branches here.

Reply:
it worked, good eye. resubmitted.